src/general/requiredlabels/src.cel

```diff
@@ -0,0 +1,5 @@
+validations:
+- expression: '[object, oldObject].exists(obj, obj != null && has(obj.metadata) && variables.params.labels.all(entry, has(obj.metadata.labels) && entry.key in obj.metadata.labels))'
```

Comment:
```suggestion
- expression: '(has(request.operation) && request.operation == "DELETE") ||(has(object.metadata) && variables.params.labels.all(entry, has(object.metadata.labels) && entry.key in object.metadata.labels))'
              messageExpression: '"object missing required label, requires all of: " + variables.params.labels.map(entry, entry.key).join(", ")'
```
This works for audit since the `||` handles `operation == ""`